src/building-blocks/FormField.tsx

```diff
@@ -45,18 +42,11 @@ class FormField extends Component<IProps> {
     }
 
     return (
-      <Info key={fieldConfig.field}>
-        <Label className={cx({ 'form-item-required': fieldConfig.required })}>
-          {fieldConfig.label}
-        </Label>
-        <Value>
-          <Antd.Form.Item>
-            {getFieldDecorator(fieldConfig.field, decoratorOptions)(
-              <EditComponent {...editProps} />,
-            )}
-          </Antd.Form.Item>
-        </Value>
-      </Info>
+      <Antd.Form.Item label={fieldConfig.label}>
```

Comment:
I think you should keep the `key={}` , otherwise, 👍

Reply:
fyi, I'm using this in the Registry:
`<Antd.Form.Item key={`field-config-${fieldConfig.field}-${index}`} label={fieldConfig.label}>`